### DISCUSSION TITLE
can any one help me with this problem in lab2?

### DISCUSSION THREAD

**Discussion by chengyu-123** [bol-edu/HLS-SOC-Discussions#55] [General]
In "3. 附錄：Integrated Logic Analyzer (ILA)" can't drag fir_n11_maxi_0 to "nets to debug", does anyone has any idea
![messageImage_1694866603463](https://github.com/bol-edu/HLS-SOC-Discussions/assets/69706789/0aa6cf5e-3234-4fef-8a77-10223afbb4fa)

**Comment by boledulab** [comment]
The ILA debug only works while pynq/kv260 is connected to your local machine which running Vivado.
It is not working on current OnlineFPGA pynq/kv260.
https://docs.xilinx.com/r/en-US/ug936-vivado-tutorial-programming-debugging/Target-Board-and-Server-Set-Up
